dist/tools/whitespacecheck/ignore_list.txt

```diff
@@ -1,2 +1,3 @@
 */vendor/*
 dist/tools/fixdep/fixdep.c
+*.svg
```

Comment:
If you use `git add -p` and then hit `a` for the file, the whitespaces are removed if you use the following git config:

```gitconfig
[apply]
	whitespace = fix
[core]
	whitespace = blank-at-eol,blank-at-eof,trailing-space,space-before-tab
```

As neither tabs nor trailing whitespaces have a syntactical meaning in SVG files, I really don't see a reason why those should be ignored.